src/Commands/PullStock.php

```diff
@@ -0,0 +1,55 @@
+<?php namespace Daalder\Exact\Commands;
+
+use Illuminate\Console\Command;
+use Illuminate\Database\Eloquent\Builder;
+use Illuminate\Support\Facades\Bus;
+use Pionect\Daalder\Models\Product\Repositories\ProductRepository;
+
+class PullStock extends Command
+{
+    /**
+     * The console command name.
+     *
+     * @var string
+     */
+    protected $signature = 'exact:pull-stock {sku?}';
```

Comment:
Can you change this to `{sku?*}` to support multiple skus?
And change the where statement to whereIn.